app/controllers/application_controller/policy_support.rb

```diff
@@ -195,7 +195,7 @@ def protect_build_tree
 
   # Create policy assignment audit record
   def protect_audit(pp, mode, db, recs)
-    msg = _("[%{name}] Policy Profile %{mode} (db:[%{db}]") % {:name => pp.name, :mode => mode, :db => db}
+    msg = "[#{pp.name}] Policy Profile #{pp.mode} (db:[#{db}]"
```

Comment:
@mzazrivec `pp.mode` should be just `mode`

Reply:
@h-kataria fixed!